src/frontend/Session.elm

```diff
@@ -139,3 +144,39 @@ fetchDocs author project version =
   Http.get
     (Url.absolute [ "packages", author, project, V.toString version, "docs.json" ] [])
     (Decode.list Docs.decoder)
+
+
+
+-- ELM.JSON
+
+
+getManifest : Data -> String -> String -> V.Version -> Maybe Project.PackageInfo
```

Comment:
The name for `elm.json` in the compiler is `Outline`. Can the mentions of `Manifest` be renamed to `Outline`?